doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -235,25 +236,27 @@ For the User OS, we are using the same `Clear Linux`_ release version as the Ser
 
   .. code-block:: none
 
-     cd ~
-     wget https://download.clearlinux.org/releases/24040/clear/clear-24040-kvm.img.xz
-     unxz clear-24040-kvm.img.xz
+     cd /root/
```

Comment:
This ``cd /root/`` (and file download and extraction) doesn't match with the file location mention on line 256.  (It would also require you to be root user to write files in ``/root``.)  I suspect this should go back to being ``cd ~``.  Right @gvancuts ?  Is your approval of this PR still valid?